llvm/test/Transforms/LoopSimplify/pr51735-1.ll

```diff
@@ -0,0 +1,120 @@
+; RUN: opt -passes=indvars -S -o - < %s | FileCheck %s
```

Comment:
These tests should be in the IndVarSimplify directory, not the LoopSimplify directory.